superset-frontend/src/explore/constants.ts

```diff
@@ -45,6 +45,7 @@ export enum Operators {
   LATEST_PARTITION = 'LATEST_PARTITION',
   IS_TRUE = 'IS_TRUE',
   IS_FALSE = 'IS_FALSE',
+  DATETIME_BETWEEN = 'DATETIME_BETWEEN',
```

Comment:
Good catch, I will change it to `TEMPORAL_BETWEEN`.

Reply:
If I have an open ended range, like "100 years ago", then isn't it rather a "BEFORE" than a "BETWEEN"? So I would still reconsider renaming this to just "TEMPORAL" or something else instead of "BETWEEN".

Reply:
the `TEMPORAL` sounds like a date type rather than an operator. how about `TEMPORAL_FROM` or `TEMPORAL_PARSER` or `CACULATE_TEMPORAL`

Reply:
How do you feel about `TEMPORAL_RANGE`?

Reply:
Like this. Thanks to InteliJ, renaming is done in 1 second.